engine/src/test/scala/cromwell/engine/workflow/WorkflowActorSpec.scala

```diff
@@ -142,10 +151,20 @@ class WorkflowActorSpec extends CromwellTestKitSpec with WorkflowDescriptorBuild
     "not run Finalization actor if aborted when in MaterializingWorkflowDescriptorState" in {
       val actor = createWorkflowActor(MaterializingWorkflowDescriptorState)
       deathwatch watch actor
+      copyWorkflowLogsPrope watch dummyLogCopyRouter
       actor ! AbortWorkflowCommand
       finalizationProbe.expectNoMsg(AwaitAlmostNothing)
       deathwatch.expectTerminated(actor)
     }
+
+    "copy workflow logs in the event of MaterializeWorkflowDescriptorFailureResponse" in {
+      val actor = createWorkflowActor(MaterializingWorkflowDescriptorState)
+      deathwatch watch actor
+      actor ! MaterializeWorkflowDescriptorFailureResponse(new Exception("Workflow Materialization Failed"))
```

Comment:
I always appreciate a big friendly "DON'T WORRY THIS EXCEPTION WAS INTENDED FOR THE TESTS"